packages/cli/src/ui/AppContainer.tsx

```diff
@@ -1042,6 +1075,26 @@ Logging in with Google... Please restart Gemini CLI to continue.
     ],
   );
 
+  const dialogsVisible =
```

Comment:
nit: why did didalogVisible get moved in this file? doesn't seem like it got referenced between its old and new location.

Reply:
will revert